.circleci/config.yml

```diff
@@ -260,11 +328,21 @@ jobs:
             pyenv versions
             pyenv global 3.7.0
       - run:
-          name: build docker
+          name: Pull Submodules
```

Comment:
Why are we pulling submodules here? Just an idea, If it is because of the dockerfile, will using conditional logic in dockerfile be better or clumsy?

Reply:
yeah so I decided not to complicate things but yeah sure I can add that!

Reply:
makes sense, lets keep this open for now to get another opinion

Reply:
sorted!